torch/_dynamo/debug_utils.py

```diff
@@ -233,7 +233,7 @@ def _cuda_system_info_comment():
         cuda_version_lines = cuda_version_out.decode().split("\n")
         comment = "".join([f"# {s} \n" for s in cuda_version_lines if s not in [""]])
         model_str += f"{comment}\n"
-    except FileNotFoundError:
+    except (FileNotFoundError, subprocess.CalledProcessError):
```

Comment:
Do we still need FileNotFoundError then?

Reply:
Yeah, so actually it's a little more complex than in my summary, if nvcc simply isn't there we'll get a  `FileNotFoundError`, but in my case I have nvcc symlinked to ccache (as recommended in our dev setup guide) but no nvcc in my path.  I think it's still valid to catch both here :)